UI/Campus-UI/src/app/sing-in/sing-in.component.ts

```diff
@@ -0,0 +1,41 @@
+import {Component, OnInit} from '@angular/core';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
+import {User} from '../shared/interfaces';
+import {SignInService} from '../shared/services/sign-in.service';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'app-sing-in',
+  templateUrl: './sing-in.component.html',
+  styleUrls: ['./sing-in.component.sass']
+})
+export class SingInComponent implements OnInit {
+
+  form: FormGroup;
+
+  // constructor(private auth: SignInService, private router: Router) {
+  // }
+
+  ngOnInit(): void {
+    this.form = new FormGroup({
+      email: new FormControl(null, [Validators.required, Validators.email]),
```

Comment:
According to the API map, we use the 'login' value on the authentication endpoint, not 'email' one.

Reply:
Unfortunately we had approved UI concept,and we'll spend valuable effort to change it

Reply:
Promise me to fix it in the next sprints!